### ISSUE TITLE
Button Name Audit

### ISSUE THREAD

**Issue by gauntface** [GoogleChrome/lighthouse#2213]
Minor note: not sure how to fix this audit.

> Buttons have an accessible name.
> An accessible name helps convey the purpose of a button. Without one, a screen reader will only 
> announce the word "button".

The description doesn't explain how to fix it or link out to learn more :(

From @robdodson 

> put text inside them, or give them a label, or give them an aria-label

**Comment by robdodson** [comment]
Yeah for a lot of these we have a 'learn more' page but I don't know if we've created one for buttons yet. @kaycebasques do you know if one exists?

Also, what's the rule about putting info for fixing lighthouse issues into the description? Some fixes might be a bit wordy.

**Comment by patrickhulce** [comment]
> Also, what's the rule about putting info for fixing lighthouse issues into the description? Some fixes might be a bit wordy.

I'm in favor of descriptions providing a very brief (one sentence) snippet about *why* they should care about fixing it and linking off to a "Learn more" link on /web for more detail and how to fix it.

**Comment by kaycebasques** [comment]
I don't have a reference for that audit, yet

I agree with @patrickhulce's approach

**Comment by paulirish** [comment]
@kaycebasques we can link to axe's docs for the a11y audits..

for example, the aXe extension shows this:

![image](https://cloud.githubusercontent.com/assets/39191/26120753/ab45cd84-3a26-11e7-9ac1-0475b3ceefb4.png)


And they link to https://dequeuniversity.com/rules/axe/2.1/link-name?application=axeChrome

our button-name link could be to https://dequeuniversity.com/rules/axe/2.1/button-name?application=lighthouse

wdyt? 
do you have some time to try this out in the next two days? would love to include in our 2.0.

**Comment by robdodson** [comment]
+1 I say we go for it

**Comment by kaycebasques** [comment]
I'm in Portland for a writer conference and traveling tomorrow, I don't think I can do it by tomorrow

I've been thinking about doing this for the a11y audits. Do we want all the "learn more" in all of the a11y audits to link directly to the aXe docs?

Pros:

* My docs are a regurgitation of their docs
* Reduces my workload

Cons:

* We lose analytics insight into how many people are traveling to these docs
* We lose control over the guidance. I've frankly found that some of the aXe docs are verbose and could be explained in a more user-friendly fashion

**Comment by robdodson** [comment]
I think we can get analytics by using the outbound query param that Paul mentioned. The folks at deque offered to track inbound traffic using that query param if I recall. I can circle back with them on this.

Personally I'm a fan of linking out to their docs. As you said, we're regurgitating them anyway. I think where their docs are verbose, we could contribute back to them and improve things that way.

**Comment by kaycebasques** [comment]
SGTM

**Comment by kaycebasques** [comment]
FYI I updated my comment a little, Rob, as you were posting your reply

**Comment by kaycebasques** [comment]
All of the a11y audits now link to the aXe docs
